src/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,169 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommandLineParser {
+
+	public HashMap<String, String> arguments;
```

Comment:
should be marked as final

Reply:
and could be `public Map<String, String> arguments`